client/components/AddSponsor.tsx

```diff
@@ -0,0 +1,113 @@
+import * as React from 'react';
+import styled from 'styled-components';
+import fetch from 'isomorphic-fetch';
+
+interface IAddSponsorProps {
```

Comment:
can move to `interface` at one single place for a component? I mean for Sponsor component, we can have single place for all the `interface`